### PR TITLE
fix: use sed replace yq

### PR DESCRIPTION
**Pull request by kamly**
fix: use sed replace yq

### CODE REVIEW

**Comment by Revolyssup** [comment]
@kamly you can reduce this back to 30 https://github.com/apache/apisix-docker/blob/39145b409bfc8f9a3883a6008031d3e256feeed7/.github/workflows/apisix-docker-example-test-standalone.yaml#L63C1-L63C1

File: centos/docker-entrypoint.sh [hunk 1]
```diff
@@ -31,11 +31,8 @@ deployment:
     config_provider: yaml
 _EOC_
       else
-          wget -qO /usr/local/apisix/yq https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 \
-	        && chmod a+x /usr/local/apisix/yq
-          /usr/local/apisix/yq -i '.deployment.role = "data_plane"' ${PREFIX}/conf/config.yaml
-          /usr/local/apisix/yq -i '.deployment.role_data_plane.config_provider = "yaml"' ${PREFIX}/conf/config.yaml 
-          rm /usr/local/apisix/yq
+          sed -i 's/role: traditional/role: data_plane/' config.yaml
```
Comment (Revolyssup) [inline centos/docker-entrypoint.sh:34]: I dont think we need to match only when role: traditional and config_provider is etcd.
It should be replaced in any case which is role: * and config_provider: *

**Review by Revolyssup — CHANGES_REQUESTED** [review]

**Comment by Revolyssup** [comment]
@kamly please fix the ci

File: .github/workflows/apisix-docker-example-test-standalone.yaml [hunk 2]
```diff
@@ -60,7 +60,7 @@ jobs:
 
       - name: Test APISIX
         run: |
-          sleep 50 #startup time maybe a little longer as the yq binary is downloaded for use at runtime
```
Comment (Revolyssup) [inline .github/workflows/apisix-docker-example-test-standalone.yaml:63]: can you also remove the comment starting from `#` ? @kamly

**Review by Revolyssup — CHANGES_REQUESTED** [review]

File: .github/workflows/apisix-docker-example-test-standalone.yaml (hunk 2 above)
Comment (Revolyssup) [reply .github/workflows/apisix-docker-example-test-standalone.yaml:63]: Also it's an urgent issue so we would want this to be merged today

**Comment by kamly** [comment]
fix it ~

**Review by Revolyssup — APPROVED** [review]

**Review by lingsamuel — APPROVED** [review]

**Review by monkeyDluffy6017 — APPROVED** [review]

**Pull request merged**